scripts/mkcloudhost/generate-radvd-conf

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+cloudfunc=~/cloud.d/cloudfunc
+test -e $cloudfunc || cloudfunc=$(dirname $(readlink -e $BASH_SOURCE))/cloudfunc
+. $cloudfunc
+
+host_letter=$(hostname | cut -b5)
+
+snippet=$(mktemp)
+cat <<EOF >$snippet
+interface v${host_letter}<NUM>br
+{
+  AdvSendAdvert on;
+  MinRtrAdvInterval 30;
+  MaxRtrAdvInterval 100;
+  AdvManagedFlag off;
+  AdvOtherConfigFlag off;
+  prefix <PREFIX>::/64
+  {
+    AdvOnLink on;
+    AdvAutonomous on;
+    AdvRouterAddr off;
+  };
+  RDNSS <PREFIX>::1
+  {
+    AdvRDNSSLifetime 300;
```

Comment:
Any reason for keeping AdvRDNSSLifetime to 300 seconds? This default to 2 * MaxRtrAdvInterval and that is the max suggested value, here: https://linux.die.net/man/5/radvd.conf

Reply:
I was lazy, and when 30 didn't work, I added a zero. I'll set it to 60.